dev.hdos.HDOS.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>dev.hdos.HDOS</id>
+    <name>HDOS</name>
+    <developer_name>HDOS</developer_name>
+    <summary>HDOS OSRS Client</summary>
+    <metadata_license>CC0-1.0</metadata_license>
```

Comment:
no project license?

Reply:
Not directly. The ptlroject itself it kept closed for now until approved otherwise by the game developers.

Reply:
>  I imagine distribution is fine.

I would say it is not given there is no information.

Reply:
Is this required?

Reply:
I'll commit CC BY-NC-ND 4.0 when I'm home.

Reply:
CC BY-NC-ND would be very unusual for source code. But yes, the project needs some license that grants Flathub redistribution rights.

Reply:
> 

is putting in <project_license>LicenseRef-proprietary</project_license> acceptable? similar to how other closed-source applications are included.

Reply:
@hfiguiere added a license, how does it look?